lib/modules/manager/sbt/extract.ts

```diff
@@ -36,6 +41,8 @@ interface Ctx {
   variableName?: string;
 }
 
+const SBT_MVN_REPO = 'https://repo1.maven.org/maven2';
```

Comment:
This does not looks correct. Is this really `repo1`?

Reply:
Yes, this is from the sbt documentation where the default resolvers are described. Surprised me too :)
https://www.scala-sbt.org/1.x/docs/Resolvers.html#Predefined+resolvers